src/views/DailyJournal/DayView/ProjectGroupedView/ContractGroupedView/WorkItemRow/index.tsx

```diff
@@ -64,13 +73,26 @@ function workItemStatusKeySelector(item: WorkItemStatusOption) {
 function workItemStatusLabelSelector(item: WorkItemStatusOption) {
     return item.label;
 }
+function workItemStatusColorSelector(item: WorkItemStatusOption): [string, string] {
+    if (item.key === 'DOING') {
+        return colorscheme[1];
+    }
+    if (item.key === 'DONE') {
+        return colorscheme[5];
+    }
+    return colorscheme[7];
```

Comment:
Better to use a function to get value from colorscheme, in case we run into out of index bound